### PR TITLE
Migrate from Alpine to Debian

### PR DESCRIPTION
**Pull request by mvorisek**
Related with #1 - test manually before closing this issue

### CODE REVIEW

**Comment by mvorisek** [comment]
@romaninsh can you please migrate https://github.com/atk4/image/blob/master/Dockerfile#L3 to `apt`?

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM php:7.4-alpine
+FROM php:7.4-buster
```
Comment (DarkSide666) [inline Dockerfile:1]: shouldn't this file go in `7.4` subfolder ?

Comment (mvorisek) [reply Dockerfile:1]: I did not move it - I assume that this means something like `latest` - check the GH Actions for ex. in the `atk4/ui` repo - there are workflows for `7.2`, `7.3` and `latest`.

So I expect, when this line will be changed in future to `8.0`, the workflow for `latest` will use it (thus it makes sense to keep it like it is (or duplicate it to offer `7.4` beside `latest` as well)).

Comment (DarkSide666) [reply Dockerfile:1]: OK. Probably you're right. Let @romaninsh check this.

**Comment by romaninsh** [comment]
Alpine is much more suitable, smaller and is a standard when it comes to sicker images. Why migrate?

**Comment by mvorisek** [comment]
@romaninsh see description - #1, Alpine has a lot of limitation/incompatibilities. Debian/Ubuntu is defacto standard for larger projects for these reasons. What is the reason to not do so? If we can boot Debian/Ubuntu in about the same time, what is the disadvantage?

**Comment by mvorisek** [comment]
@romaninsh do you know also about debian slim? If Alpine has not the required package, you are in trouble, but with slim, which should be equal to alpine, you can install any standard Debian package.

**Pull request closed without merging**